test_situations.md

```diff
@@ -0,0 +1,7 @@
+# How to test your changes on "ready to use" situations (for OpenFisca-France)
+
+Often, when making changes to legislation, you need to test it on a situation that works with your country Tax & Benefit system.
```

Comment:
> Add a link that defines a situation?
ex : http://openfisca.org/doc/openfisca-web-api/input-output-data.html#describing-the-situation